deploy/central/tasks-chart/templates/task-ods-deploy-helm.yaml

```diff
@@ -12,8 +12,9 @@ spec:
     Helm has the plugins `helm-diff` and `helm-secrets` installed. A diff is performed
     before an upgrade is attempted. `helm-secrets` can be used to encrypt sensitive
     values in the underlying Git repository. Secrets are decrypted on the fly if the secret
-    identified by the `private-key-secret` parameter exists and contains a private key
-    matching a fingerprint listed in `.sops.yaml`.
+    identified by the `age-key-secret` parameter exists, and contains an age secret key
+    which public key was used as one of the recipients to encrypt. The age-key file location
+    is known via `SOPS_AGE_KEY_FILE` environment variable (e.g.: `SOPS_AGE_KEY_FILE=/sops/age/keys.txt`)
```

Comment:
If I understand right, `SOPS_AGE_KEY_FILE` can be used when you want to customize the path, otherwise the default path is taken and the env var is not set. 
See this part of the doc: https://github.com/mozilla/sops#encrypting-using-age . It does not clearly state it, but if you do `echo $SOPS_AGE_KEY_FILE` locally nothing comes back.

Reply:
that is under construction still, that comment there explains the current approach just for the task in here: https://github.com/opendevstack/ods-pipeline/blob/4d895dbacce0458f4bd2c48fa8cb8ce5ac3b6a3e/deploy/central/tasks-chart/templates/task-ods-deploy-helm.yaml#L77

anyhow, I am not sure if that will be the final approach yet